src/screens/wallet/token-list/token-list.tsx

```diff
@@ -72,6 +79,18 @@ export const TokenList: FC = () => {
     [isShowTezos, filteredAssetsList]
   );
 
+  const screenFillingItemsCount = flatlistHeight / ITEM_HEIGHT;
+
+  const renderData =
+    isAndroid && screenFillingItemsCount > flatListData.length
+      ? flatListData.concat(Array(Math.ceil(screenFillingItemsCount - flatListData.length)).fill(emptyToken))
```

Comment:
i think "fake" tokens should have unique token slugs to let `FlatList` works correct